Makefile

```
@@ -115,6 +115,11 @@ gendependabot:
 		  echo "  - package-ecosystem: \"docker\"\n    directory: \"$${dir:1}\"\n    schedule:\n      interval: \"daily\"" >> ${DEPENDABOT_PATH} ); \
 	done
 
+.PHONY: generate
+generate:
```

Does this make sense?
```suggestion
generate: install-tools
```

---

oh duh that's how to do it, thank you!